xml/System.Reflection/Assembly.xml

```diff
@@ -4231,7 +4241,17 @@ To load the correct assembly, it's recommended to call the `Load` method by pass
         <param name="rawSymbolStore">A byte array that contains the raw bytes representing the symbols for the assembly.</param>
         <summary>Loads the assembly with a common object file format (COFF)-based image containing an emitted assembly, optionally including symbols for the assembly.</summary>
         <returns>The loaded assembly.</returns>
-        <remarks>
+        <remarks FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;net-5.0;net-6.0;net-7.0;net-8.0;net-9.0">
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This API is present in .NET Core for compatibility, but it is recommended to use <xref:System.Reflection.AssemblyLoadContext> overloads in .NET Core 2.0 and later.
+
+The assembly is loaded into the current AssemblyLoadContext.
```

Comment:
Add a link to https://learn.microsoft.com/en-us/dotnet/core/dependency-loading/loading-managed#algorithm ?